src/main/java/touro/chess/Square.java

```diff
@@ -1,5 +1,5 @@
 package touro.chess;
-
+/**
```

Comment:
Why have you done this?

Reply:
Oh I didn't mean to push that! For some reason was having issues with this import: `import org.jetbrains.annotations.Nullable;` It was easier to comment out that class for now so I could run my tests. 
Should I uncomment and push that class again?

Reply:
Yes. You cannot comment out code just because it doesn't work for you.